hw/dv/dpi/usbdpi/usbdpi.c

```diff
@@ -529,9 +529,9 @@ void pollRX(struct usbdpi_ctx *ctx, int sendHi, int nakData) {
       ctx->data[1] = 0x82;
       ctx->data[2] = 0 | CRC5(0x82, 11) << 3;
       ctx->data[3] = USB_PID_DATA0;
-      ctx->data[4] = 0x48;
-      ctx->data[5] = 0x69;
-      ctx->data[6] = 0x21;
+      ctx->data[4] = 0x48;  // "H"
```

Comment:
Couldn't this just be `ctx->data[4] = 'H';`?

Reply:
It felt easier to leave the packet assembly using hex constants as are used everywhere else in the file, but in this case provide some more documentation as to what the packet did.